.gitignore

```diff
@@ -9,6 +9,9 @@ bin/
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# For Konflux
+!vendor/github.com/go-errors/errors/cover.out
```

Comment:
This could even be `!vendor/**` as I don’t think we want to ignore anything in `vendor/`